global_logger.py

```diff
@@ -21,8 +21,10 @@
 _log_file = './logs/logon_server.log'
 
 # noinspection SpellCheckingInspection
-_log_formatter = logging.Formatter(
+_old_log_formatter = logging.Formatter(
     "%(levelname)-7.7s %(process)-12.12d %(processName)-12.12s %(threadName)-12.12s %(asctime)s: %(message)s")
+_log_formatter = logging.Formatter(
+    "%(levelname)-7.7s %(threadName)-12.12s: %(message)s")
```

Comment:
I trimmed down the log formatter to get rid of a lot of information that I didn't find very useful. I did leave the previous version renamed as _old_log_formatter.